demo/src/main/java/com/example/DemoApplication.java

```diff
@@ -31,6 +31,9 @@ public void run(String... arg0) throws Exception {
 				// Intentional bug
 				String neverUsed = "";
 
+				// Other bug
+				Integer a = 1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "a" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "a". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS1854)